app/controllers/admin/application_pets_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Admin::ApplicationPetsController < ApplicationController
+  def update
+    @application_pet = ApplicationPet.where("pet_id = ? AND application_id = ?", params[:pet_id], params[:application_id])
```

Comment:
I'd suggest `find_by` here and pass in the pet_id and application_id because it will return just one record rather than an array!